### PR TITLE
Netdata Agent: Backup restore documentation

### PR DESCRIPTION
**Pull request by luisj1983**
##### Summary

A document on how to backup and restore the Netdata agent for the public docs.
This works in a test environment and I've used it once in production so please check it over as it may need ammendments.

<details> <summary>For users: How does this change affect me?</summary>
- Public-facing documentation change
</details>

### CODE REVIEW

File: docs/maintenance/backup-restore.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!--
+title: "Backing up Netdata"
+description: "Guide on how to backup and restore Netdata."
+custom_edit_url: "https://github.com/netdata/netdata/edit/master/docs/maintenance/backup-restore.md"
+sidebar_label: "Notify"
+learn_status: "Published"
+learn_rel_path: "Integrations/Notify"
+-->
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:8]: This is not needed, but it doesn't break anything either, remnants of an older time :smile: 
```suggestion
```

File: docs/maintenance/backup-restore.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+# Introduction
+When preparing to backup Netdata it is worth considering that there are different sorts of data that you may wish to backup independently or all together:
+
+| Data type      | Description | Location |
+| ----------- | ----------- | ----------- |
+| Agent configuration| Files controlling configuration of the Netdata agent | /etc/netdata/
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:15]:
```suggestion
| Agent configuration| Files controlling configuration of the Netdata Agent | [config directory](https://github.com/netdata/netdata/blob/master/docs/configure/nodes.md#the-netdata-config-directory)
```

File: docs/maintenance/backup-restore.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+| Metrics   | Database files | /var/cache/netdata |
+| Identity   | Cloud-claim, API key and some other files | /var/lib/netdata |
+
+<br>
+Note: Only you can decide how to perform your backups. The following examples are provided as a guide only.
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:20]:
```suggestion
> ###Note
> Only you can decide how to perform your backups. The following examples are provided as a guide only.
```

File: docs/maintenance/backup-restore.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+# Introduction
+When preparing to backup Netdata it is worth considering that there are different sorts of data that you may wish to backup independently or all together:
+
+| Data type      | Description | Location |
+| ----------- | ----------- | ----------- |
+| Agent configuration| Files controlling configuration of the Netdata agent | /etc/netdata/
+| Metrics   | Database files | /var/cache/netdata |
+| Identity   | Cloud-claim, API key and some other files | /var/lib/netdata |
+
+<br>
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:19]:
```suggestion
```

If you do two line breaks I think it doesn't need a br!

File: docs/maintenance/backup-restore.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+Note: Only you can decide how to perform your backups. The following examples are provided as a guide only.
+
+
+<br>
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:23]:
```suggestion
```

**Review by Ancairon — COMMENTED** [review]
Hello @luisj1983 :wave:!

I am taking a look at the document, and will make any changes that might be needed as I read through it. I added some suggestions but I think it is easier to shoot my edits in a commit.

Thanks for this contribution!

**Comment by tkatsoulas** [comment]
@luisj1983 Added your current change in learns stagging to have an idea on how it will look 

If you want to test your changes locally 

1. Clone netdata learn  https://github.com/netdata/learn
2. Checkout to staging1 branch (you don't have permission to push on the remote branch)
3. Follow https://github.com/netdata/learn?tab=readme-ov-file#prerequisites
4. Fetch new changes
     1. `python ingest/ingest.py --repos luisj1983/netdata:maintenance_backups`
     2. `python3 ingest/create_grid_integration_pages.py `
5. run `yarn start`

File: docs/maintenance/backup-restore.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  If you wish to minimize the gap in metrics caused by stopping the Netdata Agent, then you could have a backup job or script that uses the following sequence:
+  
+- Backup the Agent configuration Identity directories
+- Stop the Netdata service
+- Backup up up the database files
+- Restart the netdata agent.
+
+### Restoring Netdata
+
+1. Ensure that the Netdata agent is installed and is [stopped](https://github.com/netdata/netdata/blob/master/docs/configure/start-stop-restart.md)
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:45]:
```suggestion
1. Ensure that the Netdata agent is installed and is [stopped](https://github.com/netdata/netdata/blob/master/packaging/installer/README.md#maintaining-a-netdata-agent-installation)
```

File: docs/maintenance/backup-restore.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    > sudo rm -Rf /var/cache/netdata
+    > ```
+
+2. Restore the backup from the archive
+
+    ```
+    sudo tar -xvpzf /path/to/netdata_backup.tar.gz -C /
+    ```
+
+3. [Start the Netdata agent](https://github.com/netdata/netdata/blob/master/docs/configure/start-stop-restart.md)
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:66]:
```suggestion
3. [Start the Netdata agent](https://github.com/netdata/netdata/blob/master/packaging/installer/README.md#maintaining-a-netdata-agent-installation)
```

File: docs/maintenance/backup-restore.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+- Backup the Agent configuration Identity directories
+- Stop the Netdata service
+- Backup up up the database files
+- Restart the netdata agent.
+
+### Restoring Netdata
+
+1. Ensure that the Netdata agent is installed and is [stopped](https://github.com/netdata/netdata/blob/master/docs/configure/start-stop-restart.md)
+
+   If you plan to deploy the Agent and restore a backup on top of it, then you might find it helpful to use the [`--dont-start-it`](https://learn.netdata.cloud/docs/installing/one-line-installer-for-all-linux-systems#other-options) option upon installation.
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:47]:
```suggestion
   If you plan to deploy the Agent and restore a backup on top of it, then you might find it helpful to use the [`--dont-start-it`](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kickstart.md#other-options) option upon installation.
```

File: docs/maintenance/backup-restore.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
+# Backing up a Netdata Agent
+
+## Introduction
+
+When preparing to backup a Netdata Agent it is worth considering that there are different kinds of data that you may wish to backup independently or all together:
+
+| Data type           | Description                                          | Location                                                                                                                |
+|---------------------|------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| Agent configuration | Files controlling configuration of the Netdata Agent | [config directory](https://github.com/netdata/netdata/blob/master/docs/configure/nodes.md#the-netdata-config-directory) |
```
Comment (Ancairon) [inline docs/maintenance/backup-restore.md:9]:
```suggestion
| Agent configuration | Files controlling configuration of the Netdata Agent | [config directory](https://github.com/netdata/netdata/blob/master/docs/netdata-agent/configuration.md) |
```

**Review by Ancairon — COMMENTED** [review]
Fixing some links, and @stelfrag to check the functionality of this PR.

**Review by tkatsoulas — APPROVED** [review]

**Pull request merged**